cmd/terramate/cli/cli.go

```diff
@@ -770,18 +769,9 @@ func (c *cli) runOnStacks() {
 		stacks[i] = e.Stack
 	}
 
-	logger.Trace().
-		Msg("Get command to run.")
-	cmdName := c.parsedArgs.Run.Command[0]
-	args := c.parsedArgs.Run.Command[1:]
-	cmd := exec.Command(cmdName, args...)
-	cmd.Stdin = c.stdin
-	cmd.Stdout = c.stdout
-	cmd.Stderr = c.stderr
-
 	logger.Trace().
 		Msg("Get order of stacks to run command on.")
-	order, reason, err := terramate.RunOrder(c.root(), stacks, c.parsedArgs.Changed)
+	order, reason, err := run.Sort(c.root(), stacks, c.parsedArgs.Changed)
```

Comment:
```suggestion
	orderedStacks, reason, err := run.Sort(c.root(), stacks, c.parsedArgs.Changed)
```

```suggestion
	sortedStacks, reason, err := run.Sort(c.root(), stacks, c.parsedArgs.Changed)
```

Overall I started to get a little confused with just "order", like it was not clear it is an actual list of stacks, but ordered, but not sure, maybe it is just me being stupid, as usual disregard at will.